CONTRIBUTING.md

```diff
@@ -54,9 +54,15 @@ We also follow these project-specific guidelines:
   constructors to all AutoValue classes to prevent classes in other packages
   from extending them.
 
+## Building instrumentation-java
+
+Run `./gradlew clean assemble check` on OS X or Linux, or run
```

Comment:
Note, you can use 'build' instead of 'assemble check'. I also mostly skip the 'clean' step, except for a last check before getting review.